x-pack/plugins/security_solution/public/common/mock/kibana_react.ts

```diff
@@ -66,61 +62,38 @@ export const createUseUiSettingMock = () => <T extends unknown = string>(
 export const createUseUiSetting$Mock = () => {
   const useUiSettingMock = createUseUiSettingMock();
 
-  return <T extends unknown = string>(
```

Comment:
I removed these generics as they were a) wrong and b) not necessary since these are mocks; consumers can modify/cast these as necessary to fit their need.

Reply:
One other note about these `uiSettings` mocks: they really shouldn't be needed, but we have >80 tests that rely on specific values being returned and I haven't yet had time to fix them individually.